src/Admin/src/RoutesDelegator.php

```diff
@@ -16,12 +28,33 @@ public function __invoke(ContainerInterface $container, string $serviceName, cal
         /** @var Application $app */
         $app = $callback();
 
-        $app->route(
-            '/admin[/{action}[/{uuid}]]',
-            AdminController::class,
-            [RequestMethodInterface::METHOD_GET, RequestMethodInterface::METHOD_POST],
-            'admin'
-        );
+        $app->get('/admin/add', GetAdminFormHandler::class, 'admin::get-admin-form');
```

Comment:
```suggestion
        $app->get('/admin/create-admin', CreateAdminFormHandler::class, 'admin::create-admin-form');
```
Since handler name is with create, for consistency, I would replace in the route `add` with `create`.